### PR TITLE
Allow None to be passed for geoms and bboxes on features

### PR DESCRIPTION
**Pull request by moradology**
This PR addresses #55 by setting the default value of `bbox` and `geometry` on features to `None`

### CODE REVIEW

File: geojson_pydantic/features.py [hunk 1]
```diff
@@ -16,10 +16,10 @@ class Feature(GenericModel, Generic[Geom, Props]):
     """Feature Model"""
 
     type: str = Field("Feature", const=True)
-    geometry: Geom
+    geometry: Geom = None
```
Comment (geospatial-jeff) [inline geojson_pydantic/features.py:19]:
```suggestion
    geometry: Optional[Geom] = None
```

Comment (moradology) [reply geojson_pydantic/features.py:19]: I agree that's easier to read. Do you think I should remove this line and incorporate the suggested change? https://github.com/developmentseed/geojson-pydantic/blob/54755500e5f5fce386b2be3e9cccb00dd830dbc1/geojson_pydantic/features.py#L12

Comment (moradology) [reply geojson_pydantic/features.py]: I opted for a via media: https://github.com/developmentseed/geojson-pydantic/pull/56/files#diff-a4fab2efe48317b94ba18803db374650b7685147f52df92c7998ad4176e4f675R12-R19

Comment (geospatial-jeff) [reply geojson_pydantic/features.py]: Whoops my apologies I forgot that the `Optional` was rolled into the `TypeVar`, i'd prefer it to remain in the `TypeVar`.  My bad! 😬

**Review by geospatial-jeff — APPROVED** [review]

**Comment by moradology** [comment]
I think this is likely good to go. Are there any other outstanding changes we really want to see prior to another release? (Getting things lined up to push a release for stac-fastapi)

**Review by vincentsarago — APPROVED** [review]

**Comment by vincentsarago** [comment]
@moradology @geospatial-jeff I'm getting a pre-commit error over #62 related to change made here 

```
geojson_pydantic/features.py:19: error: Incompatible types in assignment (expression has type "None", variable has type "Geom")
[68](https://github.com/developmentseed/geojson-pydantic/runs/6689482310?check_suite_focus=true#step:5:69)
Found 1 error in 1 file (checked 6 source files)
```

**Comment by moradology** [comment]
This looks a bit like a bug, but perhaps there's some wrinkle to the behavior here I'm missing. As you can see, the type variable is declared to be an `Optional`, which ought to include `None`:
https://github.com/developmentseed/geojson-pydantic/blob/ad3aabe87b4b9aef545ce2f8f1b8f259ea8d4bcf/geojson_pydantic/features.py#L12

I'm happy to cut a PR or somesuch to make modifications that will avoid this issue. Do we have a sense about what the right approach is?

**Comment by vincentsarago** [comment]
@moradology To be honest I not sure where this comes from so I'll be more than happy if you can have a look 🙏

**Comment by vincentsarago** [comment]
@moradology you can ignore ☝️ a other PR (https://github.com/developmentseed/geojson-pydantic/pull/64) seems to work 👌 so I guess it's just a mypy configuration issue in #62

**Pull request merged**